src/PostProcessor.cpp

```diff
@@ -242,6 +242,27 @@ FRAGMENT_SHADER_END
 "}																			\n"
 ;
 
+static const char* orientationCorrectionShader =
+SHADER_VERSION
+"#if (__VERSION__ > 120)													\n"
+"# define IN in																\n"
+"# define OUT out															\n"
+"# define texture2D texture													\n"
+"#else																		\n"
+"# define IN varying														\n"
+"# define OUT																\n"
+"#endif // __VERSION __														\n"
+"IN mediump vec2 vTexCoord;													\n"
+"uniform sampler2D Sample0;													\n"
+"OUT lowp vec4 fragColor;													\n"
+"																			\n"
+"void main()																\n"
+"{																			\n"
+"    fragColor = texture2D( Sample0, vec2(1.0 - vTexCoord.x, 1.0 - vTexCoord.y));       \n"
```

Comment:
This option explained as "Enable to render everything upside down"
You invert X coord as well. Is it not only upside down but also inside out?